jena-fuseki2/jena-fuseki-ui/package.json

```diff
@@ -40,6 +42,7 @@
     "@cypress/code-coverage": "^3.12.4",
     "@cypress/vue": "^6.0.0",
     "@eslint/js": "^9.2.0",
+    "@types/codemirror": "^5.60.15",
```

Comment:
CodeMirror is used as we import & call some of its functions in our code (for query editor). We also import the `@types` lib for codemirror, but that's development dev only.

qs is the query string parser.